packages/flutter/lib/src/material/text_field.dart

```diff
@@ -188,6 +188,54 @@ class _TextFieldSelectionGestureDetectorBuilder extends TextSelectionGestureDete
 /// ```
 /// {@end-tool}
 ///
+/// ## Reading values
+///
+/// A common way to read a value from a TextField is to use the [onSubmitted]
+/// method, which is called when the user finishes editing and receives the
```

Comment:
method => callback. This callback is applied to the text field's current value when the user finishes editing.